client/components/bcaap-form-sections/application-dashboard.tsx

```diff
@@ -113,22 +136,31 @@ export const ApplicationDashboard: React.FC<any> = () => {
         <>
           <div className='w-full border py-4 px-8 mb-2'>
             Filter By:
-            <div className='grid grid-cols-3 gap-1'>
+            <div className='grid grid-cols-4 gap-1'>
               <input
                 type='text'
                 className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
-                placeholder={'Facility Name'}
-                onChange={e => setState(p => ({ ...p, searchFacilityName: e.target.value }))}
-                value={searchFacilityName}
+                placeholder='Application Type'
+                onChange={e => setState(p => ({ ...p, searchApplicationType: e.target.value }))}
+                value={searchApplicationType}
               />
+
               <input
                 type='text'
                 className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
-                placeholder={'Confirmation ID'}
+                placeholder='Confirmation ID'
                 onChange={e => setState(p => ({ ...p, searchConfirmationID: e.target.value }))}
                 value={searchConfirmationID}
               />
 
+              <input
+                type='text'
+                className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
+                placeholder='Total Estimated Cost'
+                onChange={e => setState(p => ({ ...p, searchTotalCost: e.target.value }))}
+                value={searchTotalCost}
+              />
+
               <div className='grid grid-cols-2 gap-1'>
                 <Button onClick={handleFilter} variant='primary'>
                   <FontAwesomeIcon icon={faFilter} className='h-4 mr-2' />
```

Comment:
doesn't seem like the clear filter button works, page remains blank

Reply:
It'll refresh the table when you click on `Clear filter`

Reply:
I mean, if a user decides to manually delete the filter input instead of clicking Clear Filter, there's no way to get back to the full application table without a refresh

Reply:
Yep, I saw that. Sorry, I didn't clue in at first. I fixed it by disabling buttons when there's no input